### ISSUE TITLE
The  screen capture of  iOS devices was not clear enough.

### ISSUE THREAD

**Issue by qq386169040** [nwlunatic/ios-minicap#5]
Hi, Thank you for your great work.

I had followed the steps in Readme.md, and I already had got the  screen capture of  iOS devices. However, the screen capture was not clear enough. I had tried on two devices, iPhone 6s and iPad Air 2. The results was the same.

How can I get a clear screen capture? 

Here is the screen capture.

![2016-12-02 11 09 26](https://cloud.githubusercontent.com/assets/5905216/20821737/f9a9dff2-b87f-11e6-9fd2-becd1c351a26.png)

**Comment by nwlunatic** [comment]
Hi! thank you.
I've tested it with iPhone 6s and capture was fine. Can you please paste `run.sh` output, so i can try to reproduce?

**Comment by qq386169040** [comment]
The `run.sh` output:
```
7**************************************e
EnableDALDevices
2016-12-02 13:46:18.683 ios_minicap[10843:2530150] Available devices:
2016-12-02 13:46:18.684 ios_minicap[10843:2530150] 7**************************************e
2016-12-02 13:46:18.684 ios_minicap[10843:2530150] 0x1a11000005ac8509
resolution: 400x600
Allocating 731648 bytes for JPEG encoder
== Banner ==
version: 1
size: 24
pid: 10843
real width: 400
real height: 600
desired width: 400
desired height: 600
orientation: 
quirks: 1
banner: 1185b2a009010058200901005820001
New client connection
```
My iPhone 6s is 9.3.1 and iPad Air 2 is 10.0.2.
My xcode version is 7.3.1, cmake version is 3.7.0 and OS X 10.11.6

**Comment by nwlunatic** [comment]
@qq386169040 can you, please, tell the version of libjpeg-turbo installed? For example using
```
$ brew list libjpeg-turbo
```

and also check if [QuickTime mirroring](https://medium.com/@patrickswong/how-to-mirror-your-iphone-with-a-lightning-cable-using-quicktime-in-os-x-yosemite-ace3f40ef483#.n0dl3kkh4) looks fine

**Comment by qq386169040** [comment]
@nwlunatic 

The version of libjpeg-turbo installed is 1.4.0. The QuickTime mirroring is fine.

Thank you! Now I have solved it by install libjpeg-turbo 1.5.1.

**Comment by nwlunatic** [comment]
Thank you, i'll modify README
